lmnet/lmnet/metrics/object_keypoint_similarity.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import numpy as np
+
+
+# kpt_oks_sigmas is from: https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocotools/cocoeval.py
+kpt_oks_sigmas = np.array([.26, .25, .25, .35, .35,
+                           .79, .79, .72, .72, .62, .62, 1.07, 1.07, .87, .87, .89, .89]) / 10.0
+variances = (kpt_oks_sigmas * 2) ** 2
+
+
+def compute_oks_batch(joints_gt, joints_pred, image_size=(160, 160)):
```

Comment:
`compute_oks_on_batch` is better?

Reply:
`compute_object_keypoint_similarity_on_batch`.. I'm not sure is it too long?..

Reply:
It might be possible to just name as `compute_object_keypoint_similarity` for this method and rename current `compute_oks` to internal method like `_compute_oks`. (I believe that just batched version would be enough for API, other TF functions like `tf.layers.convolution2d` provides batched API.)